### ISSUE TITLE
Feature request: Warning: you indented lines by spaces instead of TAB when used "<<-"

### ISSUE THREAD

**Issue by andras-tim** [koalaman/shellcheck#892]
#### For new checks and feature suggestions
- [ ] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [ ] I searched through https://github.com/koalaman/shellcheck/issues and didn't find anything related

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash
cat - <<-EOF
    foo
EOF
```

#### Here's what shellcheck currently says:
No issues detected!

#### Here's what I wanted or expected to see:
I don't know exactly, but the `<<-` uses over `<<` for suppresses leading tabs.
*http://tldp.org/LDP/abs/html/here-docs.html#EX71*
